content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,259 @@
+  the information for the :guilabel:`Identification No`, :guilabel:`Passport No`, and
+  :guilabel:`Place of Birth` fields. If the employee is :guilabel:`Disabled` or a
+  :guilabel:`Nonresident`, click the check box next to the respective fields.
+- :guilabel:`Dependant`: If the employee has any dependants, that information is entered here. Type
+  in the :guilabel:`Number of Children` the employee has, and check the boxes next to
+  :guilabel:`Disabled Children` and/or :guilabel:`Other Dependent People` if applicable.
+- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
+  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
+  Using the calendar module, select the :guilabel:`Visa Expire Date` to enter the expiration date.
+
+  .. image:: new_employee/personal-info.png
+     :align: center
+     :alt: Enter any personal information for the employee in this tab.
+
+HR settings tab
+---------------
+
+This tab provides various fields for different information, depending on the country the company is
+located. Different fields are configured for different locations, however some sections appear
+regardless.
+
+- :guilabel:`Status`: If applicable, select a :guilabel:`Related User`, :guilabel:`Job Position`,
+  and :guilabel:`Language` with the drop-down menus. Type in the :guilabel:`Registration Number of
+  the Employee`, and the :guilabel:`NIF country code` if available.
+- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
+  Card` information here.
+- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
+  when the employee is working at a work center. This value affects the employee's pay, and may also
+  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
+  amount.
+- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
+  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
+  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
+  :guilabel:`Badge ID` to create a badge ID.
+
+  .. image:: new_employee/hr-settings.png
+     :align: center
+     :alt: Enter any information prompted in this tab for the employee.
+
+Personal documents tab
+----------------------
+
+This tab is only displayed for certain countries. If this tab is not visible, it is not applicable
+to the company's location.
+
+Add a file for the employee'ss :guilabel:`ID Card Copy`, :guilabel:`Driving License`,
+:guilabel:`Mobile Subscription Invoice`, :guilabel:`SIM Card Copy`, and :guilabel:`Internet
+Subscription Invoice` by clicking the :guilabel:`Upload Your File` button next to the corresponding
+field. File types that can be accepted are jpg, png, and pdf.
```

Comment:
Im a fan of using the :file: tag for file extensions, this is optional but adds visual variation to the doc!
```suggestion
field. File types that can be accepted are :file:`.jpg`, :file:`.png`, and :file:`.pdf`.
```